selftests/unit/test_loader.py

```diff
@@ -140,6 +142,7 @@ class LoaderTest(unittest.TestCase):
     def setUp(self):
         self.loader = loader.FileLoader(None, {})
         self.queue = multiprocessing.Queue()
+        self.tmpdir = tempfile.mkdtemp(prefix="avocado-test-loader")
```

Comment:
This is probably minor, but we are using `tempfile.mkdtemp(prefix='avocado_' + __name__)` everywhere else.